presto-main/src/test/java/com/facebook/presto/cost/TestComparisonStatsCalculator.java

```diff
@@ -160,7 +160,7 @@ public void setUp()
                 .setOutputRowCount(1000.0)
                 .build();
 
-        types = ImmutableMap.<Symbol, Type>builder()
+        types = TypeProvider.copyOf(ImmutableMap.<Symbol, Type>builder()
```

Comment:
TypeProvider could have a builder.

Reply:
I don't think it's worth it. It's a one-off use case and TypeProvider will eventually be removed (see comment above).